### ISSUE TITLE
Improve the activation of hidden plans

### ISSUE THREAD

**Issue by cdesquilbet** [octo-technology/epic-legacy-code-challenge#3]
As explained in the **facilitator’s Readme** (https://github.com/octo-technology/epic-legacy-code-challenge/blob/master/server/README.md#enable-a-hidden-plan), the modification of activated items requires to **stop and restart the server**.

It would be better not to patch the code nor restart the server to update the available items.

**Comment by jak78** [comment]
Yes, we should move the inventory generator parameters to configuration.json file to get auto reload
